lib/gris/caching.rb

```diff
@@ -0,0 +1,21 @@
+module Gris
+  module Caching
+    extend ActiveSupport::Concern
+
+    class_methods do
+      def cache_key_for(id)
```

Comment:
:) 

IMO let's keep it private and the footprint ever so slightly smaller if we don't think we'll directly call the method.